docs/examples/aws.md

```diff
@@ -0,0 +1,132 @@
+<!--[metadata]>
++++
+title = "Provision AWS EC2 Instances"
+description = "Using Docker Machine to provision hosts on AWS"
+keywords = ["docker, machine, cloud, aws"]
+[menu.main]
+parent="cloud_examples"
+weight=2
++++
+<![end-metadata]-->
+
+# Amazon Web Services (AWS) EC2 example
+
+Follow along with this example to create a Dockerized <a href="https://aws.amazon.com/" target="_blank"> Amazon Web Services (AWS)</a> EC2 instance (AMI).
```

Comment:
Nitpick: An EC2 instance and the AMI that uses aren't synonymous.  I'd suggest dropping the `(AMI)` bit, since there are other (unrelated) ways to create an AMI.